docs/docs/tutorials/securing-pages-and-api-routes.md

```diff
@@ -42,7 +42,7 @@ export default function Page() {
 
 ### Next.js (Middleware)
 
-With NextAuth.js 4.2.0 and Next.js 12, you can now protect your pages via the middleware pattern more easily. If you would like to protect all pages, you can create a `middleware.js` file in your root `pages` directory which looks like this:
+With NextAuth.js 4.2.0 and Next.js 12, you can now protect your pages via the middleware pattern more easily. If you would like to protect all pages, you can create a `middleware.js` file in your root directory which looks like this:
```

Comment:
Nice catch, just copied the verbiage from the [Next.js docs](https://nextjs.org/docs/advanced-features/middleware#using-middleware) to make it a bit more clear.

```suggestion
With NextAuth.js 4.2.0 and Next.js 12, you can now protect your pages via the middleware pattern more easily. If you would like to protect all pages, you can create a `middleware.js` file at the root or in the src directory (same level as your `pages`) which looks like this:
```

Also could you make this same change to the tutorial in the `versioned_docs/version-beta` version of this tutorial? We plan on making those the main docs soon. 

Thanks! :pray: